src/projects/EnsureThat/ExceptionMessages.cs

```diff
@@ -11,6 +11,7 @@ public static class ExceptionMessages
         public static string Collections_ContainsKey_Failed { get; } = "Key '{0}' does not exist.";
         public static string Collections_HasItemsFailed { get; } = "Empty collection is not allowed.";
         public static string Collections_SizeIs_Failed { get; } = "Expected size '{0}' but found '{1}'.";
+        public static string Collections_ItemDoesNotExist { get; } = "Item {0} does not exist.";
```

Comment:
Where it's being used as-is, this would perhaps need to be renamed to `Enum_` since it's not a message in the Collections arg class. However, I suggest instead to generalize the source site to be Collection instead.

It seems the desire here is to have an inverse form of the following (whether it be Enum or just any collection -- so we can extend the intent from Enum to all `Contains(T)` collections) :
```
// Note:: I tried to use Ensure {collection} .Contains, and noted we don't have that yet :-) #71 past me suggests it was added :-S Pretending it's here since y'all know what I mean
Ensure.That(new[] { 1, 2, 3 }).Contains(inputValue);
```
... meaning, the 'inverse form' would be:
```
Ensure.That(inputValue).IsIn(new[] { 1, 2, 3 })
```

👍 makes sense to me and reads more fluidly to have the assertee input before the allowlist